Code/Source/Map/MapBus.h

```diff
@@ -0,0 +1,69 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+#pragma once
+
+#include <AzCore/std/string/string.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Math/Vector3.h>
+#include <AzCore/Math/Transform.h>
+#include <AzCore/Interface/Interface.h>
+
+namespace ROS2
+{
+    class MapRequests
+    {
+    public:
+        AZ_RTTI(MapRequests, "{a95c07bd-d8d6-4e24-b089-e96d34d79ebb}");
+        virtual ~MapRequests() = default;
+
+        //! Convert Transform to the map coordinate system.
+        //! @param transform - Transform to be converted.
+        //! @return Copy of a Transform in the map coordinate system.
+        virtual AZ::Transform ConvertToMapCoordinateSystem(AZ::Transform transform) = 0;
+
+        //! Convert Transform from the map coordinate system.
+        //! @param transform - Transform to be converted.
+        //! @return Copy of a Transform in the world coordinate system.
+        virtual AZ::Transform ConvertFromMapCoordinateSystem(AZ::Transform transform) = 0;
```

Comment:
here and above, if transforms will not be changed in the overriding functions it will be better to use `const AZ::Transform &` as arguments